ls1intum/Artemis
# `Development`: Remove `@private` annotations in TS

### Checklist
#### General


#### Client

### Motivation and Context


The `@private` tag [should generally not be used](https://typedoc.org/tags/private/) in JSDoc comments. It has no effect and is only noise. I cannot find the comment anymore, but @Strohgelaender initially told me about this in his review of #6861.

### Description

I removed the `@private` tag everywhere. I also added the real `private` function signature part where applicable and fixed a client test that failed because of this.

### Steps for Testing

Check in the code that only comments, tests and obvious small things are changed. Maybe also deploy the PR to a test server and click around for a bit.

### Review Progress






#### Performance Review
no changes
#### Code Review
#### Manual Tests

### Test Coverage






does not apply
